data/mods/gen9ssb/moves.ts

```diff
@@ -105,6 +105,7 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 			this.attrLastMove('[still]');
 		},
 		onHit(pokemon) {
+			// According to bug reports, this also burns the opponent.
```

Comment:
I’m pretty sure the “bug” was anfield being active

Reply:
ok